bitcoinspv/types/config.go

```diff
@@ -0,0 +1,20 @@
+package types
+
+// GetValidNetParams returns a map of valid Bitcoin network parameters
+func GetValidNetParams() map[string]bool {
+	return map[string]bool{
+		BtcMainnet.String(): true,
```

Comment:
move it to `types/btc` package And rename it so we call `btc.GetValidBackends`

Reply:
Renamed